### ISSUE TITLE
Penis

### ISSUE THREAD

**Issue by boxy-boy** [Sonnet-Discord/sonnet-py#58]
penis

**Comment by Saragon4005** [comment]
No

**Comment by xFuney** [comment]
Banning this user from creating future issues if possible to prevent un-necessary issues and potential vandalism.

**Comment by xFuney** [comment]
Update: If user contacts me and wants to open a new issue, due to the minor nature of this issue, I'll unblock them and allow them to continue to contribute.
